server/monitor.go

```diff
@@ -102,7 +102,11 @@ func (s *Server) HandleConnz(w http.ResponseWriter, r *http.Request) {
 		case bySubs:
 			pairs[i] = Pair{Key: client, Val: int64(client.subs.Count())}
 		case byPending:
-			pairs[i] = Pair{Key: client, Val: int64(client.bw.Buffered())}
+			len := 0
+			if !client.internal {
+				len = client.bw.Buffered()
```

Comment:
Right. The compiler will do the right thing, but better name it something different.